packages/components/src/button/style.scss

```diff
@@ -65,49 +61,39 @@
 
 	&.is-primary {
 		background: color(theme(button));
-		border-color: color(theme(button) shade(20%)) color(theme(button) shade(25%)) color(theme(button) shade(25%));
-		box-shadow: inset 0 -1px 0 color(theme(button) shade(25%));
+		border-color: color(theme(button));
 		color: $white;
 		text-decoration: none;
-		text-shadow:
-			0 -1px 1px color(theme(button) shade(30%)),
-			1px 0 1px color(theme(button) shade(30%)),
-			0 1px 1px color(theme(button) shade(30%)),
-			-1px 0 1px color(theme(button) shade(30%));
+		text-shadow: none;
 
 		&:hover,
 		&:focus:enabled {
-			background: color(theme(button) shade(5%));
-			border-color: color(theme(button) shade(50%));
+			background: color(theme(button) shade(10%));
+			border-color: color(theme(button) shade(10%));
 			color: $white;
 		}
 
-		&:hover {
-			box-shadow: inset 0 -1px 0 color(theme(button) shade(50%));
-		}
-
 		&:focus:enabled {
 			box-shadow:
-				inset 0 -1px 0 color(theme(button) shade(50%)),
 				0 0 0 1px $white,
-				0 0 0 3px color(theme(button) shade(5%));
+				0 0 0 3px color(theme(button));
 		}
 
 		&:active:enabled {
 			background: color(theme(button) shade(20%));
-			border-color: color(theme(button) shade(50%));
-			box-shadow: inset 0 1px 0 color(theme(button) shade(50%));
+			border-color: color(theme(button) shade(20%));
+			color: $white;
 		}
 
+		// [aria-disabled="true"] is not specific enough to override default styling,
+		// so double-up selector to increase specificity.
 		&:disabled,
 		&:disabled:active:enabled,
-		&[aria-disabled="true"],
+		&[aria-disabled="true"][aria-disabled="true"],
```

Comment:
This one concerns me a bit. WIn general we should be extremely careful when we increase the specificity of anything at all. Especially because this generic component is used in quite a few places. 

Can you recall _where_ this specificity was insufficient? Perhaps I can find a different way to make it work?

Reply:
Yep, if you start a new post and (without having made any changes) tab over to the 'Publish' button, you'll see the issue. The hover style from the non-disabled styling is overriding this and making the button look clickable.